vignettes/spatial_wwinference.Rmd

```diff
@@ -108,20 +108,21 @@ library(ggh4x) # for facet_grid2
 # Generating Data
 
 We start by generating the data that our models require, using the
-`generate_simulation_data` function.  The presets that are required to use the
+`generate_simulation_data()` function.  The presets that are required to use the
```

Comment:
```suggestion
`generate_simulated_data()` function.  The presets that are required to use the
```

I will look up how to link this, I just tested and it didn't seem to work properly